linera-views/src/rocksdb.rs

```diff
@@ -152,7 +160,7 @@ impl KeyValueStoreClient for RocksdbClientInternal {
 /// A shared DB client for RocksDB implementing LruCaching
 #[derive(Clone)]
 pub struct RocksdbClient {
-    client: LruCachingKeyValueClient<RocksdbClientInternal>,
+    client: LruCachingKeyValueClient<KeyValueStoreClientBigValue<RocksdbClientInternal>>,
```

Comment:
I am against that approach:
* The cache is about symbolic data. So, the higher the symbol that can be stored, the better.
* If we had N-chunks, it would be N entries in the cache which would flood it.
* The leftover chunks would pollute the cache as well.
* It is true that this exposes a potential memory problem in the LRU cache, but this has to be addressed there and explicitly so.

Reply:
Oh sorry for that. I did not mean to be perceived so negatively. It was just that on this point, I felt sure while on other issues of that PR, I have far less certainties.